internal/services/oracle/validate/db_system.go

```diff
@@ -0,0 +1,185 @@
+package validate
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"unicode"
+	"unicode/utf8"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-09-01/dbsystems"
+)
+
+func DbSystemName(i interface{}, k string) (warnings []string, errors []error) {
+	v, ok := i.(string)
+	if !ok {
+		errors = append(errors, fmt.Errorf("expected type of %s to be string", k))
+		return
+	}
+
+	if len(v) < 1 || len(v) > 255 {
+		errors = append(errors, fmt.Errorf("name must be %d to %d characters", 1, 255))
```

Comment:
please define variables for `1` and `255` which are used more than once
```suggestion
		errors = append(errors, fmt.Errorf("`name` must be %d to %d characters", minLength, maxLength))
```